tools/cpp/windows_cc_configure.bzl

```diff
@@ -394,5 +398,19 @@ def configure_windows_toolchain(repository_ctx):
       "%{dbg_content}": "",
       "%{link_content}": "",
       "%{cxx_builtin_include_directory}": "\n".join(escaped_cxx_include_directories),
-      "%{coverage}": "",
-  })
+      "%{coverage}": ""
+  }
+
+
+def configure_windows_toolchain(repository_ctx):
+  """Configure C++ toolchain on Windows."""
+  repository_ctx.symlink(Label("@bazel_tools//tools/cpp:BUILD.static"), "BUILD")
+
+  vc_path = find_vc_path(repository_ctx)
+
+  all_options = _get_toolchain_options(repository_ctx, vc_path, "x64_windows", initialize_symlinks=True)
```

Comment:
Instead of passing an architecture string, pass a bool variable. That's safer to work with because there's no risk of mistyping the value you compare it to.

The only place where you use the architecture name is in line R381; there it's OK to do `"x64_windows" if is64bit else "msys_x86_32"`.

Reply:
ping

Reply:
What about ARM support? Then a bool wouldn't be enough.

Reply:
@laszlocsomor Yeah, what about (possible) ARM support? Unless we don't have `enum`s in Skylark I would like to leave it as `string`.